docker/builder-platform/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Nuxeo platform builder image
+#
+# It provides:
+# - A Nuxeo server distribution with appropriate permissions in the /distrib directory.
+# - A set of pre-installed packages on the server distribution
+# - An install-packages.sh script to install Nuxeo packages.
+#
+# It is based on an OpenJDK image as it requires Java for the package installation.
+#
+# It must be used within a multi-stage build.
+# For instance, you can use the following Dockerfile sample to build an image containing a Nuxeo Platform distribution:
+# - In the NUXEO_HOME directory.
```

Comment:
Do you mean "Build from the NUXEO_HOME directory"?

Reply:
Nop, I don't think so. The distribution is in `NUXEO_HOME` in the final image of the sample:
```
COPY --from=builder --chown=UID:GID /distrib NUXEO_HOME
```